src/main/java/Tractor.java

```diff
@@ -0,0 +1,27 @@
+public class Tractor implements Drivable, Tradable{
+    private double maxSpeed;
+
+    public Tractor() { this.maxSpeed = 100;}
+
+    @Override
+    public void upgradeSpeed() {
+        this.maxSpeed++;
+    }
+
+    @Override
+    public void downgradeSpeed() {
+        this.maxSpeed--;
+    }
+
+    @Override
+    public int getMaxSpeed() {
+        return this.maxSpeed;
+    }
+
+    @Override
+    public int getPrice() {
+        return 1000;
+    }
+
```

Comment:
Consider overriding the toString function, as in the Horse class example, and including the maximum speed of the Tractor.